README.md

```diff
@@ -52,19 +52,32 @@ https://godoc.org/github.com/go-debos/debos/actions
 
 Official debos container is available:
 ```
-docker pull godebos/debos
+$ docker pull godebos/debos
 ```
-
 See [docker/README.md](https://github.com/go-debos/debos/blob/master/docker/README.md) for usage.
 
 ## Installation (under Debian)
 
-    sudo apt install golang git libglib2.0-dev libostree-dev qemu-system-x86 \
+    $ sudo apt install golang git libglib2.0-dev libostree-dev qemu-system-x86 \
          qemu-user-static debootstrap systemd-container
-    export GOPATH=/opt/src/gocode # or whatever suites your needs
-    go get -u github.com/go-debos/debos/cmd/debos
+    $ export GOPATH=/opt/src/gocode # or whatever suites your needs
+    $ go get -u github.com/go-debos/debos/cmd/debos
     /opt/src/gocode/bin/debos --help
 
+## Build Docker images from source (Using Dockerfile)
+
+```
+$ git clone https://github.com/go-debos/debos.git
+cd debos
+$ docker build -f docker/Dockerfile -t godebos/debos
+```
+or just run:
+
+```
+$ ./build-docker.sh
```

Comment:
TBH, I am not sure about the benefit of adding the container build script. The build arguments are simple enough.

Also, as structured the change gives the impression that the build will also fetch the source code. I would suggest to split the instructions into fetching the source code and building the container.

Reply:
(but adding the docker build command to the readme is still important)